per/serializers.py

```diff
@@ -1011,6 +1021,18 @@ def validate(self, data):
             )
         return data
 
+    def validate_per(self, per):
+        if per is None:
+            raise serializers.ValidationError("This field is required")
+        country_per = list(
+            Country.objects.filter(id=self.initial_data['country']).values_list("per_overviews", flat=True)
+        )
+        if per and per.id not in country_per:
```

Comment:
We are doing this right?
```suggestion
        country_per_qs = Country.objects.filter(
            id=self.initial_data['country'],
            per_overviews=per,
        )
        if not country_per_qs.exists():
```